services/server/src/server/apiv2/verification/verification.handlers.ts

```diff
@@ -146,3 +146,31 @@ export async function verifyFromEtherscanEndpoint(
 
   res.status(StatusCodes.ACCEPTED).json({ verificationId });
 }
+
+interface VerifySimilarityRequest extends Request {
+  params: {
+    chainId: string;
+    address: string;
+  };
```

Comment:
I wonder if we should also support providing the `creationTransactionHash`. I think it makes sense if this API is called by the monitor or explorers.